File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchLiveDataUseCaseImpl.java
package com.example.simonastojanovic.memeapp.domain;

import android.arch.lifecycle.MutableLiveData;

import com.example.simonastojanovic.memeapp.model.MemesItem;

import java.util.ArrayList;

public class FetchLiveDataUseCaseImpl implements FetchLiveDataUseCase{

    private MutableLiveData<ArrayList<MemesItem>> memeLiveData = new MutableLiveData<>();

    @Override
    public MutableLiveData<ArrayList<MemesItem>> getMemeLiveData() {
        return memeLiveData;
    }
}

File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchLiveDataUseCaseImpl.kt
package com.example.simonastojanovic.memeapp.domain

import android.arch.lifecycle.MutableLiveData
import com.example.simonastojanovic.memeapp.model.MemesItem
import java.util.*


class FetchLiveDataUseCaseImpl : FetchLiveDataUseCase {

    override var memeLiveData = MutableLiveData<ArrayList<MemesItem>>()

}
